src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -700,10 +725,15 @@ public enum NuGetLogCode
         /// </summary>
         NU5123 = 5123,
 
+        /// <summary>
+        /// Warning_UnrecognizedLicenseIdentifier
+        /// </summary>
+        NU5124 = 5124,
```

Comment:
these warnings are 51, while they should've been 55.

Reply:
Are this going to stay at 51?

Reply:
I put the comment there on purpose trying to get an opinion basically. 

All of the warnings are 51, while they should've 55...should we deprecate change these warning? 
Start doing 55xx? 

I think Justin did change the warning numbers at some point. We can consider doing that in dev.